packages/lib-core/src/app/storeHandler.ts

```diff
@@ -0,0 +1,12 @@
+import type { Store } from 'redux';
+
+let store: Store;
+
+const storeHandler = {
+  setStore: (storeData: Store) => {
+    store = storeData;
+  },
+  getStore: (): Store => store,
```

Comment:
How is this conceptually different than `setUtilsConfig` ?

We could simply track reference to Redux store instance via ^^